change/@griffel-devtools-167de6fc-ff1a-4cec-a1c9-12ad30b92981.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

```suggestion
  "type": "minor",
```

Let's bump it to have a proper version

---

Updated to patch after chat :)